task-3/services/authentication.ts

```diff
@@ -0,0 +1,33 @@
+import jwt from "jsonwebtoken";
+import { StatusCodes } from "http-status-codes";
+import LoggingService from './logging';
+
+const logger = new LoggingService('Authentication');
+
+
+class AuthService {
+  static generateAccessToken = (data: any) => {
+    return jwt.sign(data, process.env.TOKEN_SECRET as string, {
+      expiresIn: "1800s",
+    });
+  };
+
+  static authenticateToken(req: any, res: any, next: any) {
+    try {
+			logger.serviceLogger('Authenticate Token', req.headers["authorization"]);
```

Comment:
There are some issues with the Code Quality.
Consider running checks and applying fixes.

Reply:
Weirdly enough it looked just fine in my IDE, I've ran perttier:fix and that issue is sorted.